proto/api_test.go

```diff
@@ -150,10 +150,10 @@ func TestCombinable(t *testing.T) {
 
 func TestSetGoErrorCopy(t *testing.T) {
 	rh := ResponseHeader{}
-	err := &Error{Message: "test123"}
-	rh.SetGoError(err)
-	err.Message = "321tset"
-	if rh.Error.Message != "test123" {
+	oErr := &Error{Message: "test123"}
+	rh.Error = oErr
+	rh.SetGoError(&testError{})
```

Comment:
Thanks for asking. Didn't realize the old test was wrong. Btw, the incoming `err` is used, but only if it's one of the `ErrorDetail` union (in which case it just gets stored in there).